server/lib/nl/common/utils.py

```diff
@@ -369,6 +389,17 @@ def get_single_date(uttr: nl_uttr.Utterance) -> types.Date:
   return None
 
 
+def get_date_range(uttr: nl_uttr.Utterance) -> types.Date:
+  classification = futils.classifications_of_type_from_utterance(
+      uttr, types.ClassificationType.DATE)
+  if (classification and isinstance(classification[0].attributes,
+                                    types.DateClassificationAttributes)):
+    # Return the first date in the classification if it is not a single date
+    if not classification[0].attributes.is_single_date:
+      return classification[0].attributes.dates[0]
```

Comment:
curious: why not return the full range instead of just the first date?

Reply:
Each Date object in DateClassificationAttributes is actually either a date range or single date. There really should only be one date in the list of dates, so for simplicity, just return the first object in the list for now.

Reply:
Interesting. I must have missed this but would be nice to have this document somewhere obvious.